### PR TITLE
Issue #785 Style guide applied to drop buttons

### PR DESCRIPTION
**Pull request by Graham-72**
backdrop/backdrop-issues#785
enlarged arrows slightly

### CODE REVIEW

**Comment by Graham-72** [comment]
Changes applied with a different PR ?

**Comment by quicksketch** [comment]
HI @Graham-72, I don't think these changes have been applied yet. The latest 1.x-new-style still has the gradient and non-uppercase letters.

**Comment by Graham-72** [comment]
Closed prematurely.

**Comment by Graham-72** [comment]
@quicksketch - with the latest 1.x-new-style the problem with the appearance of the drop button arrows seems to have disappeared and we don't need my proposed changes to its size. But we do need to modify dropbuttons.css to make the color blue and to capitalize the letters. Also we need to change style.css to remove the background gradient. It then looks like this
dropbuttondown
dropbuttonup

File: core/misc/dropbutton.css [hunk 1]
```diff
@@ -5,6 +5,14 @@
  */
 
 /**
+ * Set width of dropbuttons where extra width is needed
+ */
+#system-modules .dropbutton-wrapper div,
+#layout-edit .dropbutton-multiple div,
+#layout-list .dropbutton-wrapper div {
+  min-width: 140px;
+}
```
Comment (quicksketch) [inline core/misc/dropbutton.css:14]: The dropbutton.css file shouldn't contain any module-specific selectors like this. These should be moved to `system.admin.css` and `layout.admin.css` respectively.

Comment (Graham-72) [inline core/misc/dropbutton.css:14]: Now split into https://github.com/backdrop/backdrop/pull/853 and https://github.com/backdrop/backdrop/pull/854

**Comment by jenlampton** [comment]
Thanks @Graham-72 !  You don't need to put those changes into separate PRs, they can still be in this one, we just need to make sure the changes are to the correct files.

I'm also a little concerned as to why we have to have module-specific selectors in the first place. Why are these drop-buttons not smart enough about width that we need to do this?  Can we solve this problem with padding instead?

Also, I've closed your two most recent PRs as duplicates of this one, so let's continue working on that one until it's ready to be merged.

To update a PR:
- Checkout the branch the PR was made from.
- Make whatever changes you need to the code.
- Commit the changes with --amend to overwrite the previous commit (and if needed, commit message)
- Push the local branch up to your remote (use -f to overwrite the previous code in that branch)
- the PR should be updated right away
- feel free to add a comment stating that it's been changed.

**Comment by jenlampton** [comment]
Oh boy, @quicksketch just showed me https://github.com/backdrop/backdrop-issues/issues/370.  This looks like a doozy, I'm going to take a stab at it.

**Comment by jenlampton** [comment]
@Graham-72 I preserved your commits, and added to them.  new PR at https://github.com/backdrop/backdrop/pull/829

**Pull request closed without merging**